www/content/scoop.md

```diff
@@ -31,6 +31,9 @@ scoop:
     name: goreleaserbot
     email: goreleaser@carlosbecker.com
 
+  # The project name and current git tag are used in the format string.
+  commit_format: "chore(scoop): update {{ .Config.ProjectName }} version {{ .Git.CurrentTag }}"
```

Comment:
Should be `commit_template:`.  Also add a comment above with the default (or set the example to the default)

```yaml
  # Git commit message template used for Scoop
  # Defaults are shown.
  commit_template: "Scoop update for {{ .ProjectName }} version {{ .Version }}"
```